src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1356,10 +1356,24 @@ internal HttpRequestMessage CreateRequest<TResult>(JsonOperationContext ctx, Ser
 
                 case HttpStatusCode.Forbidden:
                     var msg = await TryGetResponseOfError(response).ConfigureAwait(false);
-                    throw new AuthorizationException("Forbidden access to " + chosenNode.Database + "@" + chosenNode.Url + ", " +
-                        (Certificate == null ? "a certificate is required. " : Certificate.FriendlyName + " does not have permission to access it or is unknown. ") +
-                        $"Method: {request.Method}, Request: {request.RequestUri}" + Environment.NewLine + msg
-                        );
+                    var builder = new StringBuilder(msg)
+                        .Append(". ").AppendLine()
+                        .Append("Forbidden access to ").Append(chosenNode.Database).Append("@").Append(chosenNode.Url).Append(", ");
+                    if (Certificate == null)
+                    {
+                        builder.Append("a certificate is required. ");
+                    }
+                    else if(Certificate.PrivateKey != null)
+                    {
+                        builder.Append(Certificate.FriendlyName).Append(" does not have permission to access it or is unknown. ");
+                    }
+                    else
```

Comment:
Those are already in the message from the server, no?

Reply:
{""Type"":""InvalidAuth"",""Message"":""This server requires client certificate for authentication, but none was provided by the client.""}

This is the message we got from the server.
I accidentally appended the `msg` twice in the beginning and in the end so I will remove it from the beginning so it will be as before.